src/support/z_context.erl

```diff
@@ -187,6 +187,7 @@ new(Req, Module) when is_map(Req) ->
     Context#context{controller_module=Module}.
 
 
+-spec set_default_language_tz(#context{}) -> #context{}.
```

Comment:
The spec in line 190 uses a record, please define a type for the record and use that instead.